pkgs/by-name/tr/tray-tui/package.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  installShellFiles,
+  nix-update-script,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "tray-tui";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "Levizor";
+    repo = "tray-tui";
+    tag = version;
+    hash = "sha256-N/ECG8J1wVoMevj3sDHiZiXIWmb6gKQFUfK0BiNHICM=";
+  };
+
+  useFetchCargoVendor = true;
+
+  cargoHash = "sha256-COz4tY6VwjpwxLNe7JIs8CBXlQyQIaDhMQkeZRKk5Zs=";
+
+  nativeBuildInputs = [
+    installShellFiles
+  ];
+
+  postInstall = ''
+    CMD=tray-tui
+    installShellCompletion --cmd $CMD\
+      --bash <($out/bin/$CMD --completions bash) \
+      --zsh <($out/bin/$CMD --completions zsh) \
+      --fish <($out/bin/$CMD --completions fish)
```

```suggestion
    installShellCompletion --cmd tray-tui \
      --bash <($out/bin/tray-tui --completions bash) \
      --zsh <($out/bin/tray-tui --completions zsh) \
      --fish <($out/bin/tray-tui --completions fish)
```
Nit: Convention is to not use variables here